src/meta-srv/src/service/admin.rs

```diff
@@ -56,26 +56,26 @@ pub fn make_admin_service(meta_srv: MetaSrv) -> Admin {
         },
     );
 
-    let router = router.route(
-        "/schemas",
-        meta::SchemasHandler {
-            table_metadata_manager: meta_srv.table_metadata_manager().clone(),
-        },
-    );
-
-    let router = router.route(
-        "/tables",
-        meta::TablesHandler {
-            table_metadata_manager: meta_srv.table_metadata_manager().clone(),
-        },
-    );
-
-    let router = router.route(
-        "/table",
-        meta::TableHandler {
-            table_metadata_manager: meta_srv.table_metadata_manager().clone(),
-        },
-    );
+    let handler = meta::SchemasHandler {
+        table_metadata_manager: meta_srv.table_metadata_manager().clone(),
+    };
+    let router = router
+        .route("/schemas", handler.clone())
+        .route("/schemas/help", handler);
```

Comment:
nit: I'm not so sure the design is graceful, but I don't really have a good idea.